### DISCUSSION TITLE
post/tech/web/instoducing-giscus-with-hugo-stack/

### DISCUSSION THREAD

**Discussion by giscus** [katsuobushiFPGA/blog-hugo-comment#1] [Announcements]
# post/tech/web/instoducing-giscus-with-hugo-stack/

http://localhost:1313/post/tech/web/instoducing-giscus-with-hugo-stack/

**Comment by katsuobushiFPGA** [comment]
テストコメント！

コードフェンスも使える
```php
<?php
  echo "Hello, World";
```
